build.sbt

```diff
@@ -155,6 +164,13 @@ lazy val core = project.in(file("core"))
   .settings(publishArtifact in(Test, packageBin) := true)
   .settings(libraryDependencies ++= (coreDependencies ++ coreTestDependencies))
 
+lazy val spark = project.in(file("spark"))
+  .settings(commonSettings: _*)
+  .settings(name := "tranquility-spark")
+  .settings(resolvers += "clojars" at "http://clojars.org/repo/")
```

Comment:
I think you don't need the clojars repo here. It's needed for clojure libs in storm, but I don't think spark uses those.